src/main/java/cat/udl/eps/softarch/learnphysics/domain/Question.java

```diff
@@ -18,7 +19,9 @@
 public class Question {
```

Comment:
Without the annotation @Entity Spring does not know this is a class of the model and you can not create relations with other classes. This is causing the application to crash.

Reply:
Check that the annotation is present in all classes of the model.